src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesPairs = sourceString.split(';');
+  const stylesArray = stylesPairs.map((pair) => {
+    return pair
+      .trim()
+      .split(':')
+      .map((part) => part.trim());
+  });
+
+  const stylesObject = {};
+
+  stylesArray.forEach(([key, value]) => {
+    stylesObject[key] = value;
```

Comment:
you can simplify it using the reduce method

Reply:
ok